src/main/java/io/opentelemetry/semconv/UserAgentAttributes.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.semconv;
+
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+
+import io.opentelemetry.api.common.AttributeKey;
+
+// DO NOT EDIT, this is an Auto-generated file from
+// buildscripts/templates/SemanticAttributes.java.j2
+@SuppressWarnings("unused")
+public final class UserAgentAttributes {
+
+  /**
+   * Value of the <a href="https://www.rfc-editor.org/rfc/rfc9110.html#field.user-agent">HTTP
+   * User-Agent</a> header sent by the client.
+   */
+  public static final AttributeKey<String> USER_AGENT_ORIGINAL = stringKey("user_agent.original");
```

Comment:
Q: do we want to have `UserAgentAttributes.USER_AGENT_ORIGINAL` or `UserAgentAttributes.ORIGINAL` ?

Reply:
> UserAgentAttributes.USER_AGENT_ORIGINAL

this is nice for statically importing the constants

though @mateuszrzeszutek recently said in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/9925#discussion_r1407425481

> static imports made this PR harder to write. If we ever decide to change things in the future again, I think I'd prefer to not have static imports here.